src/widgets/attachments/attachment-thumbnail.component.tsx

```diff
@@ -101,59 +101,55 @@ export default function AttachmentThumbnail(props: AttachmentThumbnailProps) {
           </div>
         )}
       </div>
-      <Thumbnail {...props}/>
+      <Thumbnail {...props} />
     </div>
   );
 }
 
 function ImageThumbnail(props: ImageProps) {
   return (
-    <div className={styles.imageThumbnail}>
-      <img
-        src={props.src}
-        alt={props.title}
-        style={props.style}
-      />
+    <div className={styles.imageThumbnail} role="button" tabIndex={0}>
+      <img src={props.src} alt={props.title} style={props.style} />
     </div>
-  )
+  );
 }
 
 function PdfThumbnail(props: ImageProps) {
   function handleClick(e: React.SyntheticEvent) {
     e.preventDefault();
     e.stopPropagation();
-    window.open(props.src, '_blank');
+    window.open(props.src, "_blank");
   }
 
   return (
-    <div className={styles.pdfThumbnail}>
-      <img
-        onClick={handleClick}
-        src={props.src}
-        alt={props.title}
-        style={props.style}
-      />
+    <div
+      className={styles.pdfThumbnail}
+      onClick={handleClick}
+      role="button"
+      tabIndex={0}
+    >
+      <img src={props.src} alt={props.title} style={props.style} />
     </div>
-  )
+  );
 }
 
 function OtherThumbnail(props: ImageProps) {
   function handleClick(e: React.SyntheticEvent) {
     e.preventDefault();
     e.stopPropagation();
-    window.open(props.src, '_self');
+    window.open(props.src, "_self");
```

Comment:
Why not a new window?

Reply:
Is there a better way to do this? I just want to be able to download this file and I went with this solution from searching the internet.

Reply:
But you don't know if what you open actually triggers a download. If its a TXT you'd just open it...

For downloads a better solution would be to dynamically append an `a` to the DOM with the `download` attribute. Clicking it would trigger a download (guaranteed).

Then afterwards just remove the  `a`.

Reply:
Been a while I worked on this component but the solution you proposed isn't working I guess that was why I didn't go with that approach earlier. And it has to do probably with the library used. I'm still investigating this but might take some time.

Reply:
The library event handlers seem to be overriding every other behavior and even with a 

`<a href={linkToFile} download/>` used, on clicking the file is just opened in full screen mode.